src/main/java/software/aws/neptune/jdbc/Driver.java

```diff
@@ -39,15 +41,33 @@
     public static final String DRIVER_VERSION;
     public static final String APP_NAME_SUFFIX;
     public static final String APPLICATION_NAME;
+    private static final String PROPERTIES_PATH = "/project.properties";
+    private static final String MAJOR_VERSION_KEY = "driver.major.version";
+    private static final String MINOR_VERSION_KEY = "driver.minor.version";
+    private static final String VERSION_KEY = "driver.full.version";
```

Comment:
minor nit: should we keep name consistent - `FULL_VERSION_KEY` instead of `VERSION_KEY`. and same for `version` -> `fullVersion` ?